### PR TITLE
[Governance] Custom Build Policies - Added Bicep framework

### PR DESCRIPTION
**Pull request by JBakstPaloAlto**
Jira ticket: https://redlock.atlassian.net/browse/BCE-33806

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policies.adoc [hunk 1]
```diff
@@ -6,6 +6,7 @@ Prisma Cloud includes out-of-the-box policies that enable you to detect misconfi
 You can create custom build policies for the following frameworks:
 
 * *Terraform* - Policies written using Terraform attributes will apply for Terraform (.tf and plan files).
+* *Bicep*: Policies defined using Bicep syntax for tailored Azure resource governance
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policies.adoc:9]:
```suggestion
* *Bicep*: Policies defined using Bicep resources and attributes will apply for tailored Azure Bicep resource governance.
```

I see the others are treated this way.

**Review by jenjoe22 — APPROVED** [review]
LGTM. I'd suggest a minor change in the code-editor.adoc file. Please changes all lead sentences preceding tables. The current construction is a bit awkward.
"For instructions on how to use the type of Attribute with key and values see in the table." Maybe this instead, "The table below provides instructions on how to use Attributes with matching keys and values."

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 2]
```diff
@@ -29,9 +29,13 @@ Code Editor appears as a default view.
...
 //image::governance/visual-editor-2.png
 
+. Select the target *Framework*.
++
+NOTE: Framework options include: Terraform, Kubernetes, CloudFormation and Bicep.
+
 . Select *Cloud Provider*.
 +
-You can create rules only for Cloud Service Providers that are supported on Prisma Cloud.
+You can only create rules for Cloud Service Providers that are supported on Prisma Cloud.
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:38]:
```suggestion
You can only create rules for Cloud Service Providers that are supported on Prisma Cloud via the Visual Editor. It is possible to create policies for unsupported providers using the Code Editor.
```

**Review by tsmithv11 — APPROVED** [review]

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 3]
```diff
@@ -4,85 +4,91 @@
 
 == Visual Editor
 
-Prisma Cloud supports the capability of a Visual Editor where you can create custom build policies for  your templates using the existing fields.
-The Visual Editor is a suitable option when you want to create custom policies that include Attribute checks with a support of AND/OR logic.
-To help you create a custom policy using the Visual Editor, you will see an example of custom build policy for S3 Bucket ACL where log delivery is not recommended.
-
+The 'Visual Editor' supports custom build policies for *IaC*, *Secrets* and *Licenses*, using Attribute checks that support of AND/OR logic.
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:7]: > using Attribute checks that support of AND/OR logic

I'm trying to think of how to not complicate things, but IaC supports AND/OR, Secrets and License policies only supports OR logic.

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 4]
```diff
@@ -4,85 +4,91 @@
...
-In this example, you see the policy details for S3 Bucket ACL where log delivery is not recommended.
+The *IaC Policy Code Editor* is displayed by default in the *Create query* step of the wizard. 
+//+
+//In this example, you see the policy details for S3 Bucket ACL where log delivery is not recommended.
 //+
 //image::governance/visual-editor.png
+//+
+//Code Editor appears as a default view.
+
+. Select *Visual Editor* from the menu.
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:22]:
```suggestion
. Select *Visual Editor* from the menu next to "Scan".
```

Or something to say which menu

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 5]
```diff
@@ -4,85 +4,91 @@
...
+//Code Editor appears as a default view.
+
+. Select *Visual Editor* from the menu. 
+. Select a policy type.  
+
+. Configure the policy by selecting values in the provided fields.
++
+* For Licenses, select a license type. Multiple types are supported
++
+* For Secrets, add a Regular expression. Multiple 'OR' expressions are supported
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:29]:
```suggestion
* For Secrets, add a regular expression. Multiple 'OR' expressions are supported. You can test your regular expression against a string by selecting the "Aa" button in the text field
```

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc [hunk 6]
```diff
@@ -4,85 +4,91 @@
...
 +
-Code Editor appears as a default view.
+* For IaC, see <<configure-iac,Configure Iac Custom Build Policies>> below  
++
+NOTE: Test scans are only supported for IaC policies.
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:33]: This isn't appearing as a NOTE in the preview for the page

Comment (JBakstPaloAlto) [reply docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:33]: This is an issue with asciidoc

File: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc (hunk 3 above)
Comment (JBakstPaloAlto) [reply docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc:7]: I will come up with something

**Pull request merged**